utils/windowfunction/idborderby.cpp

```diff
@@ -744,7 +744,7 @@ void IdbOrderBy::initialize(const RowGroup& rg)
     // initialize rows
     IdbCompare::initialize(rg);
 
-    uint64_t newSize = fRowsPerRG * rg.getRowSize();
```

Comment:
I'm curios why do you account the header size also? We should reuse an existing RowGroup just applying RGData to it.